src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,22 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    private List<? extends Machine> bulldozers = new ArrayList<>();

```

Comment:
redundant field

we do not need to save a new bulldozed each time the 'get' method has been called.